mlir/lib/Dialect/Rock/Pipelines/Pipelines.cpp

```diff
@@ -122,28 +123,29 @@ void rock::buildKernelPipeline(OpPassManager &pm,
    *   --rock-gemm-to-gridwise --rock-regularize
    *   --rock-gridwise-gemm-to-blockwise
    */
-  pm.addNestedPass<func::FuncOp>(rock::createRockAffixTuningParametersPass(
+  auto &funcPm = pm.nest<func::FuncOp>();
```

Comment:
Does this consolidate the passes into 1 list?

Reply:
I think it does from the point of view of printing the pass pipeline, so it shouldn't have a practical effect, but yeah.